File: archives/leetcode2/0168. Excel Sheet Column Title.py
class Solution(object):
    def convertToTitle(self, n):
        """
        :type n: int
        :rtype: str
        """
        ss="0ABCDEFGHIJKLMNOPQRSTUVWXYZ"
        if n<=26:
            return ss[n]
        rep=''
        while n>0:
            if n%26==0:
                rep='Z'+rep
                n=n//26-1
            else: 
                rep=ss[n%26]+rep
                n=n//26
        return rep

File: archives/C#/0168. Excel Sheet Column Title.cs
public class Solution {
    public string ConvertToTitle(int n) {
        string s="ABCDEFGHIJKLMNOPQRSTUVWXYZ";
        string rep="";
        while (n>0){
            if(n%26!=0){
                int tmp=n%26;
                rep=s[tmp-1]+rep;
                n=n/26;
            }
            else{
                rep='Z'+rep;
                n=(n-1)/26;
            }
        }
        return rep;
    }
}
